### DISCUSSION TITLE
Is there a way to get ignition status?

### DISCUSSION THREAD

**Discussion by vineethktpla** [flobz/psa_car_controller#821] [Q&A] [answered]
Hi,
I'm wanted to track trips for my e-2008 in home assistant, but there is no trips api endpoint available. I was tracking my trips using kinetic>moving attribute, but it's not reliable.

Is there a way to get ignition status of the car ?
Like if the car is ON or OFF

**Comment by 5kat3R** [comment, marked as answer]
In the JSON there is an attribute: **ignition / type**.
Have you tried to follow that how does it behave compared to the usage of your vehicle?
![kuva](https://github.com/flobz/psa_car_controller/assets/52790629/2823c179-0423-44fa-b887-7d547f36c2c0)

**Reply by vineethktpla** [reply]
I will check if it updates.
Thanks a lot :-)

**Reply by vineethktpla** [reply]
It works. Thanks @5kat3R

**Reply by zazazouthecat** [reply]
Exemple sensor for ignition (French)

```
# etat de la voiture (allumée ou arretée)
e208_state_vehicle:
	friendly_name: "Etat Voiture"
	value_template: >-
		{% if 'Stop' in state_attr('sensor.peugeot_e208', 'ignition')['type'] -%}
			Arretée
		{% elif 'StartUp' in state_attr('sensor.peugeot_e208', 'ignition')['type'] -%}
			Allumée
		{% else -%}
			??
		{% endif -%}
	icon_template: >-
		{% if 'Stop' in state_attr('sensor.peugeot_e208', 'ignition')['type'] -%}
			mdi:car-off
		{% elif 'StartUp' in state_attr('sensor.peugeot_e208', 'ignition')['type'] -%}
			mdi:car-electric
		{% else -%}
			mdi:help-circle-outline
		{% endif -%}
```

rename e208_state_vehicle by e2008_state_vehicle for example and rename sensor.peugeot_e208 by your e2008 sensor
